helpers/validators.js

```diff
@@ -27,8 +31,20 @@ const emailValidator = {
   }
 }
 
+const phoneOnKeyPress = (e) => {
+  const charCode = e.which ? e.which : e.keyCode;
+
+  if (charCode == 43){
+    return true
+  }
+  if (charCode > 31 && (charCode < 48 || charCode > 57)) {
```

Comment:
Is this trying to exclude characters that are not digits?

What is the relevance of `charCode > 31` ?
Remove it if it's not required.

Combining with the suggestions from the comment above:
```suggestion
  if (charCode < '0'.charCodeAt() || charCode > '9'.charCodeAt())) {
```

In addition, it could be combined to allow '+' as follows:
```suggestion
  if ((charCode < '0'.charCodeAt() || charCode > '9'.charCodeAt()) && !charCode == '+'.charCodeAt()) {
```